### DISCUSSION TITLE
Diff command fails in GH actions but not in Travis

### DISCUSSION THREAD

**Discussion by dingo-d** [community/community#27196] [Actions]
<p>I am trying to migrate the CI process from Travis to GH Actions, but one thing keeps bothering me.</p>
<p>Here: <a href="https://github.com/WPTT/WPThemeReview/pull/261/checks?check_run_id=1542166027" rel="noopener nofollow ugc">https://github.com/WPTT/WPThemeReview/pull/261/checks?check_run_id=1542166027</a></p>
<p>The diff command I’m running is showing differences.</p>
<p>The analogue build here: <a href="https://travis-ci.com/github/WPTT/WPThemeReview/jobs/450067838" rel="noopener nofollow ugc">https://travis-ci.com/github/WPTT/WPThemeReview/jobs/450067838</a></p>
<p>is not (and is passing).</p>
<p>I tried changing the OS from ubuntu-latest to ubuntu-16.04, thinking that maybe something changed in the <code>diff</code> command, but it too failed on GH Actions.</p>
<p>If anybody can help me understand why it fails I’d be grateful.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p><code>diff</code> returns 1 if the checked files are different, and has been for a long time. So I think the thing to look at is <em>why</em> the files are different in the Actions build. At a glance the differences seem to be in indentation, maybe the defaults for <code>xmllint</code> are different?</p>

**Comment by dingo-d** [comment]
<p>That was the issue. Thanks! I added <code>export XMLLINT_INDENT=$'\t'</code> before the diff run and it worked :slight_smile:</p>
